### DISCUSSION TITLE
double-lined triple-backtick breaks the whole discussion page

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46041] [Discussions]
### Select Topic Area

Bug

### Body

\```
\```
^^^
This two line breaks other lines to a single text like this:

You can test this bug on here:
https://github.com/EarlGreytea16/actionwithdiscussion/discussions/14

![Animation](https://user-images.githubusercontent.com/115380484/216143215-7461853d-ee02-43f5-91b2-9bb26a1ef2ba.gif)

You could see that under "Example Code" section's markdown breaks to text.

**Comment by LindseyB** [comment]
Is this still happening. I'm getting a 404 with the link you provided.
